src/main/java/com/juniorstart/juniorstart/payload/PagedResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.juniorstart.juniorstart.payload;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class PagedResponse<T> {
```

Comment:
Czy ta klasa nie powinna rozszerzać np. ResponseEntity?

Reply:
Znaczy mogę tak zrobić, ale czy istnieje taka potrzeba

Reply:
Hmm, zwracanie kodów statusu. Ale to niech ktoś się wypowie. Ewentualnie później do Ciebie wrócę z odpowiedzią po reaserch'u.

Reply:
Hmm , w takim razie w kontrolerze jakbym dodał :  @ResponseStatus(HttpStatus.OK)
Samo zapytanie albo zwróci coś albo nic w przypadku złych danych rzuci i tak wyjątek, który "przyćmi" kod 200 
Czy się mylę w tym co pisze.

Reply:
A czy jeśli metoda zwróci wyjątek to zwróci go ze statusem 200 czy nie?

Reply:
Nie

Reply:
ResponseEntity powinno się zwracać tylko wtedy, jeżeli endpoint nie potrzebuje zadnych danych zwrotnych np usuwanie

Reply:
Ta klasa bardzo ułatwia frontowi korzystanie z rest api